bl@0.7.0 requires a peer of stream-browserify@* but none was installed

I receive the following error when installing vinyl-map:

```
project@0.1.0 ~/project
└─┬ vinyl-map@1.0.1 
  └── UNMET PEER DEPENDENCY stream-browserify@*

npm ERR! peer dep missing: stream-browserify@*, required by bl@0.7.0
```

It seems [bl v0.7.0 has no API changes when compared to the latest version v1.1.2](https://github.com/rvagg/bl/compare/v0.7.0...v1.1.2), and a temporary update passes all tests.

Will submit a pull request with the change shortly....

--- a/package.json
+++ b/package.json
@@ -1,14 +1,15 @@
 {
   "name": "vinyl-map",
   "description": "Map vinyl files' contents as strings, so you can easily use existing code without needing yet another gulp plugin!",
-  "version": "1.0.1",
+  "version": "1.0.2",
   "main": "index.js",
   "dependencies": {
     "through2": "^0.4.1",
-    "bl": "^0.7.0",
+    "bl": "^1.1.2",
     "new-from": "0.0.3"
   },
   "devDependencies": {
+    "concat-stream": "~1.5.2",
     "tape": "~2.3.2",
     "vinyl": "~0.2.1",
     "uglify-js": "~2.4.12",

